source/reference/minio-mc/mc-event.rst

```diff
@@ -43,12 +43,12 @@ Subcommands
           :start-after: start-mc-event-add-desc
           :end-before: end-mc-event-add-desc
 
-   * - :mc:`~mc event list`
+   * - :mc:`~mc event ls`
      - .. include:: /reference/minio-mc/mc-event-list.rst
```

Comment:
Might be worth considering renaming these files, since that's how they show in the URL.
For this and the other commands.

Not critical, but worth considering.